website/static/js/dashboard.js

```diff
@@ -397,26 +441,59 @@ var FileBrowser = {
                 });
             }
             self.breadcrumbs().push(linkObject);
-        }.bind(self);
+        };
 
-        self.sidebarInit = function (element, isInit) {
-            if(!isInit){
-                $('[data-toggle="tooltip"]').tooltip();
-            }
+        // GET COLLECTIONS
+        // Default system collections
+        self.collections = m.prop([].concat(self.systemCollections));
+        self.collectionsPageSize = m.prop(5);
+        // Load collection list
+        self.loadCollections = function _loadCollections (url){
+            var promise = m.request({method : 'GET', url : url, config : xhrconfig});
+            promise.then(function(result){
+                result.data.forEach(function(node){
+                    var count = node.relationships.linked_nodes.links.related.meta.count;
+                    self.collections().push(new LinkObject('collection', { path : 'collections/' + node.id + '/linked_nodes/', query : { 'related_counts' : 'children', 'embed' : 'contributors' }, systemCollection : false, node : node, count : m.prop(count) }, node.attributes.title));
```

Comment:
Minor: This can be refactored with `.map`:

```js
self.collection(results.data.map(function() {
    //...
    return new LinkObject(...)
}))
```